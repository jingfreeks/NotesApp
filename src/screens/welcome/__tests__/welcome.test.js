import React from 'react';
import {render} from '@testing-library/react-native';
import WelcomeScreen from '../welcome';


describe('Welcome Screen', () => {
  it('Should work as expected to get snapshot', () => {
    const all = render(<WelcomeScreen />);
    expect(all.toJSON()).toMatchSnapshot();
  });
});
