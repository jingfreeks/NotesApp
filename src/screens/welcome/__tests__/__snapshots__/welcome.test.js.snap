// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Welcome Screen Should work as expected to get snapshot 1`] = `
<View
  alignType="center"
  justify="center"
  style={
    [
      {
        "alignItems": "center",
        "flexBasis": 0,
        "flexDirection": "column",
        "flexGrow": 1,
        "flexShrink": 1,
        "justifyContent": "center",
      },
    ]
  }
>
  <Text
    style={
      [
        {
          "color": "#000000",
          "fontFamily": "Avenir",
          "fontSize": 24,
          "fontWeight": "900",
        },
      ]
    }
  >
    Welcome
  </Text>
  <Text
    style={
      [
        {
          "color": "#0e3f6c",
          "fontFamily": "Avenir",
          "fontSize": 16,
          "fontWeight": "400",
        },
      ]
    }
  >
    Notes with map application
  </Text>
  <View
    style={
      [
        {
          "flexDirection": "row",
          "paddingBottom": 15,
          "paddingLeft": 15,
          "paddingRight": 15,
          "paddingTop": 15,
        },
      ]
    }
  >
    <View
      style={
        [
          {
            "borderColor": "grey",
            "borderRadius": 10,
            "borderWidth": 1,
            "flexBasis": 0,
            "flexGrow": 1,
            "flexShrink": 1,
          },
        ]
      }
    >
      <View
        accessibilityState={
          {
            "busy": undefined,
            "checked": undefined,
            "disabled": undefined,
            "expanded": undefined,
            "selected": undefined,
          }
        }
        accessibilityValue={
          {
            "max": undefined,
            "min": undefined,
            "now": undefined,
            "text": undefined,
          }
        }
        accessible={true}
        collapsable={false}
        focusable={true}
        onClick={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          {
            "alignItems": "center",
            "backgroundColor": "transparent",
            "borderRadius": 1,
            "opacity": 1,
            "paddingBottom": 15,
            "paddingLeft": 15,
            "paddingRight": 15,
            "paddingTop": 15,
          }
        }
      >
        <Text
          style={
            [
              {
                "color": "#0e3f6c",
                "fontFamily": "Avenir",
                "fontSize": 16,
                "fontWeight": "400",
              },
            ]
          }
        >
          Sign-Up
        </Text>
      </View>
    </View>
    <View
      style={
        [
          {
            "marginLeft": 5,
            "marginRight": 5,
          },
        ]
      }
    />
    <View
      style={
        [
          {
            "borderColor": "grey",
            "borderRadius": 10,
            "borderWidth": 1,
            "flexBasis": 0,
            "flexGrow": 1,
            "flexShrink": 1,
          },
        ]
      }
    >
      <View
        accessibilityState={
          {
            "busy": undefined,
            "checked": undefined,
            "disabled": undefined,
            "expanded": undefined,
            "selected": undefined,
          }
        }
        accessibilityValue={
          {
            "max": undefined,
            "min": undefined,
            "now": undefined,
            "text": undefined,
          }
        }
        accessible={true}
        collapsable={false}
        focusable={true}
        onClick={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          {
            "alignItems": "center",
            "backgroundColor": "transparent",
            "borderRadius": 1,
            "opacity": 1,
            "paddingBottom": 15,
            "paddingLeft": 15,
            "paddingRight": 15,
            "paddingTop": 15,
          }
        }
      >
        <Text
          style={
            [
              {
                "color": "#0e3f6c",
                "fontFamily": "Avenir",
                "fontSize": 16,
                "fontWeight": "400",
              },
            ]
          }
        >
          Log-In
        </Text>
      </View>
    </View>
  </View>
</View>
`;
